build-tools/xaprepare/xaprepare/ThirdPartyNotices/K4os.Compression.LZ4.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Xamarin.Android.Prepare
+{
+	[TPN]
+	class MiloszKrajewski_K4os_Compression_LZ4_TPN : ThirdPartyNotice
+	{
+		static readonly Uri url = new Uri ("https://github.com/MiloszKrajewski/K4os.Compression.LZ4/");
+
+		public override string LicenseFile => String.Empty;
+		public override string Name => "MiloszKrajewski/K4os.Compression.LZ4";
+		public override Uri SourceUrl => url;
+		public override string LicenseText => @"
```

Comment:
Why `LicenseText` and not `LicenseFile`, pulling in https://github.com/MiloszKrajewski/K4os.Compression.LZ4/blob/master/LICENSE ?

Reply:
Because this is coming in via NuGet Package.